src/java/org/apache/cassandra/index/sai/plan/QueryController.java

```diff
@@ -233,7 +253,27 @@ public UnfilteredRowIterator getPartition(PrimaryKey key, ReadExecutionControlle
             throw new IllegalArgumentException("non-null key required");
 
         SinglePartitionReadCommand partition = getPartitionReadCommand(key, executionController);
-        return executePartitionReadCommand(partition, executionController);
+        return partition.queryMemtableAndDisk(cfs, executionController);
+    }
+
+    public UnfilteredRowIterator getPartition(PrimaryKey key, ColumnFamilyStore.ViewFragment view, ReadExecutionController executionController)
```

Comment:
Looks like there is a semantic difference between this `getPrtition` and the `getPartition` above - could you document that or reflect it in the method name?

Reply:
I'll look closer and add some detail.

Reply:
I added javadocs. Let me know what you think.